app/constants/constants.js

```diff
@@ -194,6 +197,28 @@ export const CONNECTION_CONFIG = {
             'type': 'password',
             'description': 'Your data.world read/write token. It can be obtained from https://data.world/settings/advanced'
         }
+    ],
+    [DIALECTS.ATHENA]: [
+        {
+            'label': 'S3 Access Key', 'value': 'accessKey', 'type': 'password'
+        },
+        {
+            'label': 'S3 Secret Access Key', 'value': 'secretKey', 'type': 'password'
+        },
+        {
+            'label': 'AWS Region', 'value': 'region', 'type': 'text'
+        },
+        {
+            'label': 'S3 Bucket', 'value': 's3Outputlocation', 'type': 'text',
+            'description': 'The Athena will put the output results of the query in this location.'
+        },
+        {
+            'label': 'Database', 'value': 'database', 'type': 'text'
```

Comment:
What is `Database` in Athena's context?

Reply:
It is not exactly the same as a db in a normal SQL db server.  I just looked at the Athena docs and their description is pretty clear:

```
Athena uses Apache Hive to define tables and create databases, which are essentially a logical namespace of tables.

When you create a database and table in Athena, you are simply describing the schema and the location where the table data are located in Amazon S3 for read-time querying. Database and table, therefore, have a slightly different meaning than they do for traditional relational database systems because the data isn't stored along with the schema definition for the database and table.
```

Here is a link to the docs:
https://docs.aws.amazon.com/athena/latest/ug/creating-tables.html

Reply:
Oh! I thought Athena was a serverless DB. So if I understood correctly, the user has to create databases and schemas as they do for other DBs.

Again, thanks for the link.